service/hook/gogs/gogs_test.go

```diff
@@ -122,28 +122,9 @@ func Test_transformCodePushEvent(t *testing.T) {
 				},
 			},
 		}
-		hookTransformResult := transformCodePushEvent(codePush)
+		hookTransformResult := transformPushEvent(codePush)
 		require.EqualError(t, hookTransformResult.Error, "The commit specified by 'after' was not included in the 'commits' array - no match found")
-		require.False(t, hookTransformResult.ShouldSkip)
```

Comment:
I'd leave these tests here, it helps with code revisions, to ensure compatibility, or to show when these do change